[MTA 8.2.0] New consolidated prerequisites for the DL guide (#388)
migtools/mta-documentation: docs/topics/developer-lightspeed/ref_supported-large-language-models-and-providers.adoc

:_newdoc-version: 2.18.7
:_template-generated: 2026-07-10
:_mod-docs-content-type: REFERENCE

[id="supported-large-language-models-and-providers_{context}"]
= Supported large language models and providers

[role="_abstract"]

> Reviewing the supported large language models (LLMs) and providers ensures an accurate system configuration. {mta-dl-full} ({ProductShortName}) supports several options.

Author: ```suggestion
You can review the supported large language models (LLMs) and providers to ensure an accurate system configuration.
```

> The following table lists the LLM providers and example models. You can configure these in the Tackle custom resource (CR).

Reviewer: ```suggestion
To ensure an accurate system configuration, review the supported large language models (LLMs) and providers that you can configure in the Tackle custom resource (CR).
```

the two paragraphs feel like a repetition, so can we consolidate

[NOTE]
====
The availability of public LLM models is maintained by the respective LLM provider.
====

.Configurable large language models and providers
|===
| LLM Provider (Tackle CR value) | Large language model examples for Tackle CR configuration

| OpenShift AI platform | Models deployed in an OpenShift AI cluster that can be accessed by using an OpenAI-compatible API.
| OpenAI (`openai`) | gpt-4, gpt-4o, gpt-4o-mini, gpt-3.5-turbo
| Azure OpenAI (`azure_openai`) | gpt-4, gpt-35-turbo
| Amazon Bedrock (`bedrock`) | anthropic.claude-3-5-sonnet-20241022-v2:0, meta.llama3-1-70b-instruct-v1:0

> | Google Gemini (`google`) | gemini-2.0-flash-exp, gemini-1.5-pro

Reviewer: @mpershina - can you confirm that gemini-2.0-flash-exp and gemini-1.5-pro are still supported
